README.md

```diff
@@ -154,10 +154,22 @@ Provisioning script setup:
 
 Examples:
 
-- Downloads the gfx94X S3 artifacts from GitHub CI workflow run 14474448215 (from [GitHub CI workflow run 14474448215](https://github.com/ROCm/TheRock/actions/runs/14474448215)) to the default output directory `therock-build`:
+- Downloads the gfx94X S3 artifacts (except test artifacts) from GitHub CI workflow run 14474448215 (from [GitHub CI workflow run 14474448215](https://github.com/ROCm/TheRock/actions/runs/14474448215)) to the default output directory `therock-build`:
```

Comment:
These "Provisioning TheRock" instructions in the root README are getting too verbose. Concrete, copy/paste-able examples are good, but this now has 6 examples. Instead, I would structure as 1-3 canonical examples for common cases and then documentation for each individual option. You could also link to scripts/workflows to show how the setup works in practice.